gradle/javadoc-options.gradle

```
@@ -26,6 +26,9 @@
  * 2. @implSpec - the implementation specification
  * 3. @implNote - the commentaries and notes about the implementation
  *
+ * It also explicitly states the encoding of the source files from which the javadoc is composed,
+ * broadening the range of environments that the `javadoc` task is runnable under.
```

```suggestion
 * ensuring correct execution of the `javadoc` task.
```

Otherwise, it seems that there is a documented constraint on the range of environments.